source/faq.rst

```diff
@@ -0,0 +1,243 @@
+
+	.. prompt:: bash
+
+	   sudo pmm-admin add mysql --disable-tablestats
+
+	.. note:: Table statistics are disabled automatically
+	   if there are over 1 000 tables.
+
+	For more information, run as root
+	|pmm-admin.add|
+	|opt.mysql|
+	|opt.help|.
+
+.. _privileges:
+
+What privileges are required to monitor a |mysql| instance?
+================================================================================
+
+See :ref:`pmm.conf-mysql.user-account.creating`.
+
+Can I monitor multiple service instances?
+================================================================================
+
+Yes, you can add multiple instances of |mysql| or some other service to be
+monitored from one |pmm-client|. In this case, you will need to provide a
+distinct port and socket for each instance, and specify a unique name for each
+instance (by default, it uses the name of the |pmm-client| host).
+
+For example, if you are adding complete MySQL monitoring for two local |mysql|
+servers, the commands could look similar to the following:
+
+.. code-block:: bash
+
+   $ sudo pmm-admin add mysql --username root --password root instance-01 127.0.0.1:3001
+   $ sudo pmm-admin add mysql --username root --password root instance-02 127.0.0.1:3002
+
+For more information, run
+
+.. code-block:: bash
+
+   $ pmm-admin add mysql --help
+
+Can I rename instances?
+================================================================================
+
+You can remove any monitoring instance as described in :ref:`pmm-admin.rm`
+and then add it back with a different name.
+
+When you remove a monitoring service, previously collected data remains
+available in |grafana|.  However, the metrics are tied to the instance name.  So
+if you add the same instance back with a different name, it will be considered a
+new instance with a new set of metrics.  So if you are re-adding an instance and
+want to keep its previous data, add it with the same name.
+
+.. _troubleshoot-connection:
+
+How to troubleshoot communication issues between PMM Client and PMM Server?
+================================================================================
+
+Broken network connectivity may be caused by rather wide set of reasons.
+Particularly, when :ref:`using Docker <run-server-docker>`, the container is
+constrained by the host-level routing and firewall rules. For example, your
+hosting provider might have default *iptables* rules on their hosts that block
+communication between |pmm-server| and |pmm-client|, resulting in *DOWN* targets
+in Prometheus. If this happens, check firewall and routing settings on the
+Docker host.
+
+Also |pmm| is able to generate a set of diagnostics data which can be examined
+and/or shared with Percona Support to solve an issue faster. You can get
+collected logs from PMM Client using the ``pmm-admin summary`` command. 
+Obtaining logs from PMM Server can be done `by specifying the
+``https://<address-of-your-pmm-server>/managed/logs.zip`` URL, or by clicking
```

Comment:
it can be even ``https://<address-of-your-pmm-server>/logs.zip``

let's keep it simpler